assets/app/components/AddSite/TemplateSiteList/index.js

```diff
@@ -27,31 +51,50 @@ class TemplateList extends React.Component {
   render() {
     const { templates } = this.props;
 
+    const templateKeys = Object.keys(templates);
+    // if there are fewer templates than cells per row,
+    // fill the space with them
+    const cellsPerRow = Math.min(templateKeys.length, MAX_CELLS_PER_ROW);
+    // generate a two-dimensional array of template keys
+    const templateRows = createRowsOf(templateKeys, cellsPerRow);
+    // i.e. 'whole', 'half', 'third', or 'fourth'
+    const cellSize = CELL_WIDTHS[cellsPerRow];
+
+    let index = 0;
+
+    const templateGrid = templateRows.map((row, rowIndex) => {
```

Comment:
Not egregious, but a refactor point down the line may be to pull this logic out into an external `renderTemplateGrid` function. Then we can invoke it as `{this.renderTemplate(templates)}` below.